scripts/gitlab/check_runtime.sh

```diff
@@ -106,6 +109,22 @@ then
 	spec_version or or impl_version have changed in substrate after updating Cargo.lock
 	please make sure versions are bumped in polkadot accordingly
 	EOT
+
+	# Now check if any of the substrate changes have been tagged B2-breaksapi
+	(
+		cd "${SUBSTRATE_CLONE_DIR}"
+		substrate_changes="$(sanitised_git_logs "${SUBSTRATE_PREV_REF}" "${SUBSTRATE_NEW_REF}")"
+		echo "$substrate_changes" | while read -r line; do
+			pr_id=$(echo "$line" | sed -E 's/.*#([0-9]+)\)$/\1/')
+
+			if has_label 'paritytech/substrate' "$pr_id" 'B2-breaksapi'; then
+				boldprint "Substrate change labelled with B2-breaksapi. Labelling..."
+				github_label "B2-breaksapi"
+				exit 1
+			fi
+		done
+	)
```

Comment:
with the brackets that doesn't quite work as intended I guess:
```
% cat test.sh 
#/bin/sh

( exit 1 )

echo exited already
```
% ./test.sh  
exited already

Reply:
Good catch. That's what I get for using subshells! Will fix like so thoughts?:

```
#!/bin/sh

( exit 1 ) || exit 1

echo exited already
```

Reply:
I'm not sure if i understand if that subshell is really needed here. Otherwise if that works ^^ :+1: